mlflow/server/js/src/experiment-tracking/components/artifact-view-components/LazyShowArtifactAudioView.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { LegacySkeleton } from '@databricks/design-system';
+import { SectionErrorBoundary } from '../../../common/components/error-boundaries/SectionErrorBoundary';
+
+const ShowArtifactAudioView = React.lazy(() => import('./ShowArtifactAudioView'));
+
+export const LazyShowArtifactAudioView = (props: any) => (
```

Comment:
is it possible to export the props type from `ShowArtifactAudioView.tsx` and use it here?

Reply:
Done!